pushy/src/main/java/com/eatthepath/pushy/apns/ApnsClientHandler.java

```diff
@@ -507,6 +507,11 @@ public void channelInactive(final ChannelHandlerContext context) throws Exceptio
         }
 
         this.unattachedResponsePromisesByStreamId.clear();
+
+        if (!getChannelReadyPromise(context.channel()).isDone()) {
+            log.debug("Failed to create channel due to inactivity");
+            getChannelReadyPromise(context.channel()).tryFailure(STREAM_CLOSED_BEFORE_REPLY_EXCEPTION);
+        }
```

Comment:
Minor: we could simplify this to:

```suggestion
        if (!getChannelReadyPromise(context.channel()).tryFailure(STREAM_CLOSED_BEFORE_REPLY_EXCEPTION)) {
            log.debug("Failed to create channel due to inactivity");
        }
```

Reply:
It's nice. I applied. :+1: